saved-sites/saved-sites-store/src/main/java/com/duckduckgo/savedsites/store/SavedSitesEntitiesDao.kt

```diff
@@ -79,6 +79,9 @@ interface SavedSitesEntitiesDao {
     @Query("update entities set lastModified = :lastModified where entityId = :entityId")
     fun updateModified(entityId: String, lastModified: String = DatabaseDateFormatter.timestamp())
 
+    @Query("select * from entities limit 1")
+    fun lastModified(): Flow<Entity>
```

Comment:
review naming or query here. It seems we call it last modified, but it will return something random.

Reply:
good catch